docs/getting-started.md

```diff
@@ -0,0 +1,186 @@
+# Getting Started
+
+These instructions assume you use MacOS, and that you are on the internal Broad
+network or the VPN. If the VPN is not installed, follow the instructions
+[at this link](https://broad.io/vpn).
+
+> During this process, you will need your GitHub username, password, and
+personal access token for multiple steps, so make sure to have those handy. If
+you don't have those yet, see the section below, otherwise you can skip to
+[Request Required Access](#2-request-required-access).
+
+## 1. Create a GitHub account
+
+GitHub is where the Broad stores our code and projects. Sign up to this service
+with your **personal** email: https://github.com/join
+
+Create a [personal access token](https://help.github.com/en/github/authenticating-to-github/creating-a-personal-access-token-for-the-command-line)
+so you can interact with GitHub on the command line.
+
+## 2. Request Required Access
+
+Ensure that you have access to the required team resources. If you encounter a
+permission error, it is likely because you are missing appropriate access.
+
+- DataBiosphere: Join the `#github` Slack channel, click the lightning bolt in
+the channel header, and select `Join DataBiosphere`.  Once you've been granted
+access to DataBiosphere, ask a team member to add your GitHub user to the
+[jadeteam](https://github.com/orgs/DataBiosphere/teams/jadeteam) group. This
+will give you access to our repositories.
+- Google Groups: Ask a team member for access to Google Groups including
+`jade-internal` and `dsde-engineering`.
+
+## 3. Connect accounts
+
+> Make sure 2-factor authentication (2FA) is activated on your [Broad](https://broad.io/2fa)
+and [GitHub](https://github.com/settings/security) account before starting this
+process!
+
+Connect your GitHub account to your Broad profile:
+
+1. Go to [Broad people](https://people.broadinstitute.org/me) and select the
+**My Profile** tab.
+2. Link your profile to GitHub by clicking under **Other Profiles**.
+3. [Check if the account is successfully linked](https://github.broadinstitute.org/).
+4. Open each of the following GitHub groups and **Request to join** by going to
+the Members tab: [Broad Institute Read](https://github.com/orgs/broadinstitute/teams/broad-institute-read),
+[Prometheus](https://github.com/orgs/broadinstitute/teams/prometheus),
+[DSDE Engineering](https://github.com/orgs/broadinstitute/teams/dsde-engineering)
+5. To avoid being overwhelmed with notifications, [add your Broad email address](https://github.com/settings/emails),
+[route the notifications](https://github.com/settings/notifications) to that
+email, and [unfollow projects](https://github.com/watching) that are not
+relevant to your team.
+
+## 4. Create Terra Accounts
+
+The Data Catalog and [Terra](https://terra.bio/) use [Sam](https://github.com/broadinstitute/sam)
+to abtract identity and access management. To gain access to these services,
+first create a non-Broad email address through Gmail. This email address will
```

Comment:
"a non-Broad email address" -- maybe provide a suggestion here so people's accounts follow a convention? I created mine when I was helping out the monster team so mine is `monster.pshapiro.dev@gmail.com`

Reply:
I'm not sure we should suggest anything specific here. I started including this in the document, but we should support any email in general for Terra accounts. I'm concerned that any convention we set will become a standard that something will depend on.